src/components/Releases/ReleaseItem.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './ReleaseItem.css';
+
+const ReleaseItem = ({ releaseTitle, releaseId, coverArt }) => (
+  <div className={styles.releaseDiv}>
+    <h3>{releaseTitle}</h3>
+    <img src={ coverArt ? `http://coverartarchive.org/release/${releaseId}/front` : 'https://www.placecage.com/g/250/250'} alt={releaseTitle} />
```

Comment:
consider putting the different parts of the ternary on different lines for better readability